### PR TITLE
Acrolinx quality improvements3: auditpol-backup.md

### PR DESCRIPTION
**Pull request by gswashington**

### CODE REVIEW

**Comment by Jak-MS** [comment]
#assign: @robinharwood, @Xelu86 

Can you review the proposed changes? IMPORTANT: When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge.

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

File: WindowsServerDocs/administration/windows-commands/auditpol-backup.md [hunk 1]
```diff
@@ -1,20 +1,18 @@
...
 
-Backs up system audit policy settings, per-user audit policy settings for all users, and all auditing options to a comma-separated value (CSV) text file.
+The auditpol backup command backs up system audit policy settings, per-user audit policy settings for all users, and all auditing options to a comma-separated value (CSV) text file.
 
-To perform *backup* operations on the *per-user* and *system* policies, you must have **Write** or **Full Control** permission for that object set in the security descriptor. You can also perform *backup* operations if you have the **Manage auditing and security log** (SeSecurityPrivilege) user right. However, this right allows additional access that is not necessary to perform the overall *backup* operations.
+To perform *backup* operations on the *per-user* and *system* policies, you need **Write** or **Full Control** permissions for that object set in the security descriptor. You can also perform *backup* operations if you have the **Manage auditing and security log** (SeSecurityPrivilege) user right. However, this right allows additional access that isn't necessary to perform the overall *backup* operations.
```
Comment (robinharwood) [inline WindowsServerDocs/administration/windows-commands/auditpol-backup.md:15]:
```suggestion
To perform *backup* operations on the *per-user* and *system* policies, you need **Write** or **Full Control** permissions for that object set in the security descriptor. You can also perform *backup* operations if you've the **Manage auditing and security log** (SeSecurityPrivilege) user right. However, this right allows more access than necessary to perform the overall *backup* operations.
```

**Review by robinharwood — CHANGES_REQUESTED** [review]
Hey @gswashington - could you also look to simplify line 32 and replace the word additional on line 41, for example, Other references.

Thanks,

Robin

**Comment by robinharwood** [comment]
Looks good, thanks @gswashington

**Review by robinharwood — APPROVED** [review]

**Pull request merged**